### PR TITLE
kernel-install: refactor some variables

### PR DESCRIPTION
**Pull request by lnussel**
In preparation for plugin support

### CODE REVIEW

File: src/kernel-install/90-loaderentry.install.in [hunk 1]
```diff
@@ -39,13 +39,17 @@ else
...
 case "$COMMAND" in
     remove)
         [ "$KERNEL_INSTALL_VERBOSE" -gt 0 ] && \
-            echo "Removing $BOOT_ROOT/loader/entries/$ENTRY_TOKEN-$KERNEL_VERSION*.conf"
+            echo "Removing ${LOADER_ENTRY%.conf}*.conf"
         exec rm -f \
-            "$BOOT_ROOT/loader/entries/$ENTRY_TOKEN-$KERNEL_VERSION.conf" \
-            "$BOOT_ROOT/loader/entries/$ENTRY_TOKEN-$KERNEL_VERSION+"*".conf"
+            "$LOADER_ENTRY" \
+            "${LOADER_ENTRY%.conf}"*".conf"
```
Comment (daandemeyer) [inline src/kernel-install/90-loaderentry.install.in:52]: Is this bash specific syntax or regular POSIX shell syntax? We need to support regular posix shell for kernel-install.

(We really should just rewrite kernel-install in C but that's a whole different story)

Comment (keszybz) [reply src/kernel-install/90-loaderentry.install.in:52]: I think it's POSIX.

**Comment by keszybz** [comment]
jammy tests failures seem to be about `TEST-36-NUMAPOLICY`.

File: src/kernel-install/90-loaderentry.install.in (hunk 1 above)
Comment (ldv-alt) [reply src/kernel-install/90-loaderentry.install.in:52]: `${parameter%word}` is definitely POSIX.

**Pull request merged**